pkg/filter/dt/filter_http.go

```diff
@@ -30,6 +33,38 @@ import (
 
 const httpFilter = "HttpDistributedTransaction"
 
+type MatchType byte
+
+const (
+	Exact MatchType = iota
```

Comment:
`Exact` is an adjective while `Prefix` & `Regex` are noun, so how about `Equal` instead of `Exact`?